modules/openapi-generator/src/main/java/org/openapitools/codegen/templating/mustache/UncamelizeLambda.java

```diff
@@ -0,0 +1,53 @@
+
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
I think you can remove this line

ref: https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license

Reply:
I'll remove it after merging this PR.

thank again for the contribution.